### PR TITLE
DAQIANN-RESUME

### PR DESCRIPTION
**Pull request by DAQIANN**
Couldn't quite get the formatting in correctly, but this is mostly what I have on my resume. I don't have the github and linkedin links in my actual resume, Should I add this in?

### CODE REVIEW

File: resume.md [hunk 1]
```diff
@@ -1,49 +1,47 @@
...
-
-## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+**University of Illinois Urbana Champaign** *May 2023* *GPA 3.89* *Dean's List*
+* Bachelor of Science in Mathematics and Computer Science 
+* Minor in Statistics 
+* **Coursework**: Data Structures, Numerical Methods I, Software Design Studio, Discrete Structures, Introduction to Computer Science, Linear Algebra, Statistical Analysis, Statistics and Probability I
```
Comment (alextakahashi) [inline resume.md]: Would use the generic name for these classes / would remove introduction classes
* Numerical Methods I (not sure what this is)
* Software Design Studio (not sure what this is)
* ~~Introduction to Computer Science~~

File: resume.md [hunk 2]
```diff
@@ -1,49 +1,47 @@
...
-
-## AWARDS/ACKNOWLEDGEMENTS
-* **COLLEGE Alumni Scholarship Recipient** - Alumni Scholarship Committee... *Month 20XX*
-* **1st Place Best Hack** - Hackathon Event... *Month 20XX*
-
-## Events/Conferences
-* **Cal Hacks 7.0 (remote)** - Berkeley, CA ... *Month 20XX*
-* **Cal Hacks 6.0** - Berkeley, CA ... *Month 20XX*
-* **WWDC 2019** - San Jose, CA ... *Month 20XX*
+**CS 128 Course Assistant**: *September 2020 - Present*
```
Comment (alextakahashi) [inline resume.md]: Use the generic name for this - outside of your school no one knows what this is.

Would update the following
* Company --> University of Blah
* City, State --> University's City / State

^ this will also help utilizing more white space in your resume

Comment (alextakahashi) [reply resume.md]: Is there a link to online course material?

File: resume.md [hunk 3]
```diff
@@ -1,49 +1,47 @@
...
+* Course assistant for CS 128, a new core Computer Science class to be implemented Spring 2021
```
Comment (alextakahashi) [inline resume.md]: Be specific + use action verbs.  Are you a founding member?  What is your role as course assistant?  How does this highlight your technical abilities?  How many people are going to be in the course? Is this upper division?

File: resume.md [hunk 4]
```diff
@@ -1,49 +1,47 @@
...
+* Developed engaging assignments in C++ for future students
+* Worked cohesively with other assistants to produce the course material
```
Comment (alextakahashi) [inline resume.md]: This section should be pivoted to be more technical.  A bit hand wavy

File: resume.md [hunk 5]
```diff
@@ -1,49 +1,47 @@
...
+
+**Sandia National Laboratories Space Blimp Challenge**: *November 2019*
+* Led team that developed new software that improves the management of experimental flight data
```
Comment (alextakahashi) [inline resume.md:20]: +1 add who hosted the event (I'm assuming its some association of some sort) + location.

Be specific - what kind of software what kind of tools, **why** does this matter?  How does this effect business metrics by using your software?

File: resume.md [hunk 6]
```diff
@@ -1,49 +1,47 @@
...
-* **Cal Hacks 6.0** - Berkeley, CA ... *Month 20XX*
-* **WWDC 2019** - San Jose, CA ... *Month 20XX*
+**CS 128 Course Assistant**: *September 2020 - Present*
+* Course assistant for CS 128, a new core Computer Science class to be implemented Spring 2021
+* Developed engaging assignments in C++ for future students
+* Worked cohesively with other assistants to produce the course material
+
+**Sandia National Laboratories Space Blimp Challenge**: *November 2019*
+* Led team that developed new software that improves the management of experimental flight data 
+* Effectively utilized Java to parse through given data for maximum time efficiency
```
Comment (alextakahashi) [inline resume.md]: How much data?  What methods / algorithms did you use to get to where you needed to be?   How well did you perform compared to other folks in the challenge? 

Is this code on github?

File: resume.md [hunk 7]
```diff
@@ -1,49 +1,47 @@
...
+
+**Alameda County and Engineering Fair**: *April 2018*
+* Presented a research study focused on Artificial Intelligence technologies and Machine Learning to industry professionals
```
Comment (alextakahashi) [inline resume.md]: Be specific, what was the research study?  I need to better understand your personal role in this.  What did you do as an independent contributor to this project?

File: resume.md [hunk 8]
```diff
@@ -1,49 +1,47 @@
...
+* Analyzed Artificial Intelligence services such as Amazon Rekognition Services, Google Cloud Vision Service, Microsoft’s Cognitive Services Vision and Face on their image correctness, accuracy, and comprehensiveness
```
Comment (alextakahashi) [inline resume.md:25]: hand wavy, be specific to what end were you using these services?

File: resume.md [hunk 9]
```diff
@@ -1,49 +1,47 @@
...
+
+**Special Interest Group Artificial Intelligence and Data Analytics (SIGAIDA)**: *October 2019 - Present*
+* Participated in comprehensive workshops focused around the applications of Machine Learning and Artificial Intelligence within the tech world
+* Gained experience with the use of PyTorch for neural network training 
+* Managing NLP Chatbot project written in Python
+
+**Special Interest Group Programing Languages (SIGPLAN)**: *October 2019 - Present*
+* Worked with students to reinforce knowledge of several high-level programming languages and algorithms 
+
+## SKILLS
```
Comment (alextakahashi) [inline resume.md:44]: My two cents - add skills to the top this will help the reader understand what tools you are aware about.  Experience should emphasize *how* you use these tools

Comment (alextakahashi) [reply resume.md:44]: would add any version control experience you have under your belt, git

^ consider putting this into a bucketed approach

* **Programming Languages**: Java, C, ...
* **Languages**: Mandarin Chinese (Fluent)
* **Enterprise Software (?)** Office, Drive, blah


```

File: resume.md [hunk 10]
```diff
@@ -1,49 +1,47 @@
...
+* Led team that developed new software that improves the management of experimental flight data 
+* Effectively utilized Java to parse through given data for maximum time efficiency
+
+**Alameda County and Engineering Fair**: *April 2018*
+* Presented a research study focused on Artificial Intelligence technologies and Machine Learning to industry professionals
+* Analyzed Artificial Intelligence services such as Amazon Rekognition Services, Google Cloud Vision Service, Microsoft’s Cognitive Services Vision and Face on their image correctness, accuracy, and comprehensiveness 
+
+## LEADERSHIP/ACTIVITIES
+**Illinois Formula Electric**: *August 2020 - Present*
+* Member of the Data Acquisition Analysis (DAQA) subsystem
```
Comment (alextakahashi) [inline resume.md:29]: How did you contribute and why does this matter?

File: resume.md [hunk 11]
```diff
@@ -1,49 +1,47 @@
...
+* Worked collaboratively with clubmates to enhance data visualization by using Python
```
Comment (alextakahashi) [inline resume.md]: Be more specific, did you use a specific library/module to accomplish this?  How much did you improve?  Did you create graphs / diagrams?  Anything online that we can see?

File: resume.md [hunk 12]
```diff
@@ -1,49 +1,47 @@
...
+
+**Special Interest Group in Security (SIGPWNY)**: *August 2020 - Present*
+* Engaged in presentations focused around information security fundamentals
+* Competed in Capture The Flag cyber competitions
```
Comment (alextakahashi) [inline resume.md]: I don't know what this is ^ explain the technical challenges behind this

File: resume.md [hunk 13]
```diff
@@ -1,49 +1,47 @@
...
+* Presented a research study focused on Artificial Intelligence technologies and Machine Learning to industry professionals
+* Analyzed Artificial Intelligence services such as Amazon Rekognition Services, Google Cloud Vision Service, Microsoft’s Cognitive Services Vision and Face on their image correctness, accuracy, and comprehensiveness 
+
+## LEADERSHIP/ACTIVITIES
+**Illinois Formula Electric**: *August 2020 - Present*
+* Member of the Data Acquisition Analysis (DAQA) subsystem
+* Worked collaboratively with clubmates to enhance data visualization by using Python
+
+**Special Interest Group in Security (SIGPWNY)**: *August 2020 - Present*
+* Engaged in presentations focused around information security fundamentals
```
Comment (alextakahashi) [inline resume.md:33]: Vague - what does it mean to engage in presentations?

File: resume.md [hunk 14]
```diff
@@ -1,49 +1,47 @@
...
+* Competed in Capture The Flag cyber competitions
+
+**Special Interest Group Artificial Intelligence and Data Analytics (SIGAIDA)**: *October 2019 - Present*
+* Participated in comprehensive workshops focused around the applications of Machine Learning and Artificial Intelligence within the tech world
+* Gained experience with the use of PyTorch for neural network training 
+* Managing NLP Chatbot project written in Python
```
Comment (alextakahashi) [inline resume.md]: be more specific - talk more about the tools you used to make this chatbot.

**Review by alextakahashi — CHANGES_REQUESTED** [review]
Daniel - this is a good start.  I've left comments + suggestions below.

In general
* **Highlight your Technical Background**: Dive deep into your experiences about how you were critical in execution for this project.  What tools did you use?  What was the problem you were trying to solve?  How much did you improve metrics?
* **Less Broad + More Defined Experiences**:  You have a lot of experiences, pick a couple from these buckets, maybe 3 projects where you can highlight.  Be very specific

Passing back to you for revision, feel free to request another review

File: resume.md (hunk 9 above)
Comment (DAQIANN) [reply resume.md:44]: Can I pivot the Progamming Languages section to just Programming? And include git into this as I don't think I have enough to fill a separate Version Control section

**Comment by alextakahashi** [comment]
Closing in favor of a new pull request
